pycode/memilio-epidata/memilio/epidata/getCommuterMobility.py

```diff
@@ -179,14 +179,14 @@ def get_commuter_data(setup_dict='',
     try:
         population = pd.read_json(directory + "county_current_population.json")
         if len(population) != len(countykey_list):
-            population = getPopulationData.get_age_population_data(
-                out_folder=out_folder, merge_eisenach=False, write_df=True)
+            population = gPd.get_population_data(
+                out_folder=out_folder, merge_eisenach=False, read_data=read_data)
     except:
         print("Population data was not found. Download it from the internet.")
-        population = getPopulationData.get_age_population_data(
-            out_folder=out_folder, merge_eisenach=False, write_df=True)
+        population = gPd.get_population_data(
+            out_folder=out_folder, merge_eisenach=False, read_data=read_data)
```

Comment:
You're right. But I think the function should not download if read_data is True. I think it is a better idea to remove the try...except and just call get_population_data with read_data=read_data.

Reply:
I like the way where read data is set to false in the try.. except. If someone downloads the commuter data by setting read data to false but has already downloaded the population data it can be read directly from the directory. If one calls with read data set to true the commuter data is read from the local directory and if there is no population data locally I think its good to download it directly in this case.